react-native/react/menubar/index.render.desktop.js

```diff
@@ -253,41 +251,73 @@ class FolderList extends Component {
   }
 }
 
-class Footer extends Component {
-  props: {
-    debug: boolean,
-    showHelp: () => void,
-    showMain: () => void,
-    quit: () => void
-  };
-
-  render () {
-    return (
-      <div style={{display: 'flex', flexDirection: 'row', justifyContent: 'space-between', paddingLeft: 11, paddingRight: 11, paddingTop: 8, paddingBottom: 8}}>
-        <div style={{...commonStyles.clickable, ...commonStyles.transparentBlack, fontSize: 13, lineHeight: '17px'}} onClick={this.props.showHelp}>Help</div>
-        {this.props.debug && <div style={{...commonStyles.clickable, ...commonStyles.transparentBlack, fontSize: 13, lineHeight: '17px'}} onClick={this.props.showMain}>Debug</div>}
-        <div style={{...commonStyles.clickable, ...commonStyles.transparentBlack, fontSize: 13, lineHeight: '17px'}} onClick={this.props.quit}>Quit</div>
-      </div>
-    )
+const styles = {
+  container: {
+    ...globalStyles.flexBoxColumn,
+    flex: 1
+  },
+  body: {
+    ...globalStyles.flexBoxColumn,
+    ...globalStyles.rounded,
+    overflow: 'hidden',
+    height: 364,
+    minHeight: 364,
+    maxHeight: 364,
+    flex: 1
+  },
+  arrow: {
+    width: 0,
+    height: 10,
+    minHeight: 10,
+    borderLeft: '7px solid transparent',
+    borderRight: '7px solid transparent',
+    borderBottom: `5px solid ${globalColors.grey5}`,
+    alignSelf: 'center'
+  },
+  header: {
+    ...globalStyles.flexBoxRow,
+    backgroundColor: globalColors.grey5,
+    minHeight: 32,
+    maxHeight: 32,
+    padding: 10
+  },
+  icons: {
+    ...globalStyles.clickable,
+    color: globalColors.grey2
+  },
+  openingMessage: {
+    ...globalStyles.flexBoxColumn,
+    backgroundColor: globalColors.blue,
+    alignItems: 'center'
+  },
+  folderList: {
+    ...globalStyles.flexBoxColumn,
+    backgroundColor: globalColors.white,
+    flex: 1,
+    paddingTop: 17,
+    paddingLeft: 18,
+    paddingBottom: 9,
+    overflowY: 'scroll',
+    overflowX: 'hidden'
+  },
+  footer: {
+    ...globalStyles.flexBoxRow,
+    backgroundColor: globalColors.grey5,
+    justifyContent: 'space-between',
+    padding: 10
   }
 }
 
-const rootFolderStyle = {
-  fontSize: 15,
-  lineHeight: '17px',
-  color: colors.black,
-  marginBottom: 12
-}
-
 const personalTLDStyle = {
   fontSize: 15,
   lineHeight: '18px',
-  color: colors.lightBlue
+  color: globalColors.lightBlue
 }
 
 const SVGFolderIcon = svgPath => ({
   height: 28,
-  width: 25,
```

Comment:
If you just set the width , with flexbox it'll 'lose' to items with flexGrow. if you look at this in the component sheet and resize the window down you'll see you get in a state where the icons are clipped